lua/neotest-vstest/init.lua

```diff
@@ -33,6 +36,14 @@ function DotnetNeotestAdapter.root(path)
     return solution_dir
   end
 
+  if vim.g.roslyn_nvim_selected_solution then
+    solution = vim.g.roslyn_nvim_selected_solution
+    solution_dir = vim.fs.dirname(solution)
+    solution_projects = dotnet_utils.projects(solution)
+    logger.info(string.format("neotest-vstest: using solution from roslyn.nvim %s", solution))
+    return solution_dir
+  end
```

Comment:
Really nice addition! I think it would be cool to expand on this idea in the future and make it a user configurable setting to define heuristics for auto selecting a solution

Reply:
I can add a config option for this straight away if you want😄

Reply:
If it’s a quick job for you I think it would be a nice addition 🙏🏻

Reply:
Good question. It is probably better to save for another PR and spend some time to think of what it should look like!